src/Commands/DevSetupEnv.php

```diff
@@ -0,0 +1,375 @@
+<?php
+/**
+ * Copyright (c) Since 2020 Friends of Presta
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file docs/licenses/LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/afl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    Friends of Presta <infos@friendsofpresta.org>
+ * @copyright since 2020 Friends of Presta
+ * @license   https://opensource.org/licenses/AFL-3.0  Academic Free License ("AFL") v. 3.0
+ */
+
+namespace FOP\Console\Commands;
+
+use Configuration;
+use FOP\Console\Command;
+use PrestaShop\PrestaShop\Core\Crypto\Hashing;
+use ShopUrl;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class DevSetupEnv extends Command
+{
+    /**
+     * @var SymfonyStyle
+     */
+    protected $io;
+
+    /**
+     * @var mixed
+     */
+    protected $helper;
+
+    /**
+     * @var \Db
+     */
+    protected $dbi;
+
+    /**
+     * @var Hashing
+     */
+    protected $crypto;
+
+    protected $isMultiShop;
+
+    protected function configure()
+    {
+        $this->setName('fop:dev:setup-env')
+            ->setDescription('Install your project for local developement')
+            ->setHelp('<info>This command update database configuration with dev parameters (url, ssl, passwords). </info>' . PHP_EOL .
+                        '<info>How to use : </info>' . PHP_EOL .
+                        '<info>php bin/console fop:dev:setup-env --url=url.local --modify-employee-pwd=1 --modify-customer-pwd=1 --employee-pwd=fopisnice --customer-pwd=fopisnice --ssl=0</info>'
```

Comment:
```suggestion
                        '<info>php bin/console fop:dev:setup-env --host=localhost --purl="/pro/clients/jojo/"  --modify-employee-pwd=1 --modify-customer-pwd=1 --employee-pwd=fopisnice --customer-pwd=fopisnice --ssl=0</info>'
```
2 changes here : 
- `url` param is renamed `host` (must be check on other places)
- `purl` param is added